test/CMakeLists.txt

```diff
@@ -3,6 +3,17 @@
 #
 # Rahul, Jeff Ator, Ed Hartnett
 
```

Comment:
Maybe add Rahul's last name here for consistency.

Reply:
OK

Reply:
Done.